docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,60 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## User experience updates in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+
+_Generally available in Grafana Cloud_
+
+### Quickly troubleshoot with snapshot of issues
+
+Release 1.3.1. To provide quicker identification and troubleshooting, you can view a snapshot of issues on the home page that reach these specific thresholds:
+
+  - Pods that have been in a non-running state for more than 15 minutes
+  - Nodes with CPU usage above 90% for more than five minutes
+  - Nodes using more than 90% of memory for more than five minutes
+  - Persistent Volumes with capacity above 90%
+  - Node disks with capacity above 90%
+
+![Kubernetes Monitoring home page](/media/docs/grafana-cloud/k8s/K8smon-snapshotview.png)
+
+### Navigate with tabs
+
+Release 1.3.0. Quickly switch between the Cluster, namespace, workload, and Node views on the **Cluster Navigation** page by using tabs.
+
+![Tabs on Cluster navigation page](/media/docs/grafana-cloud/k8s/k8smon-clusternav-tabs.png)
+
+### Change data source on Cost and Efficiency views
+
+Release 1.2.1. You can change the data source you are viewing on the Cost and Efficiency views. 
+
+### Predict namespace memory usage
+
+Release 1.2.1. Click **Predict Memory usage** to predict namespace memory usage on the namespace detail page.
+
+![Predict Memory usage button](/media/docs/grafana-cloud/k8s/k8smon-predict-memusage-namespace.png)
+
+## Configuration updates in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+
+_Generally available in Grafana Cloud_
+
+### Configure with streamlined process
+
+Release 1.2.0. Configure with [Grafana Kubernetes Monitoring Helm chart]({{< relref "/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/config-k8s-agent-flow" >}}) using a streamlined process. With this method, you select with toggle buttons to gather metrics, logs, events, traces, and cost metrics.
```

Comment:
We get to use simple full URLs now! However, this URL did a 404 when I tested it locally.
```suggestion
Release 1.2.0. Configure with [Grafana Kubernetes Monitoring Helm chart](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/config-k8s-agent-flow) using a streamlined process. With this method, you select with switches to gather metrics, logs, events, traces, and cost metrics.
```